assets/scss/_styles_project.scss

```
@@ -1539,3 +1539,39 @@ input[type=search]::-webkit-search-cancel-button {
   text-decoration: underline;
   text-decoration-style: dotted;
 }
+
+// Changelog labels
+
+.changelog-outer {
+  display: flex;
+}
+
+.changelog div {
+  padding-right: 4px;
+}
```

I think you should be able to merge these
```suggestion
.changelog div {
  display: flex;
  padding-right: 4px;
}
```